mplc/constants.py

```diff
@@ -50,9 +51,9 @@
 NUMBER_OF_DOWNLOAD_ATTEMPTS = 3
 
 # Supported multipartner learning approaches
-MULTI_PARTNER_LEARNING_APPROACHES = [
-    "fedavg",
-    "seq-pure",
-    "seq-with-final-agg",
-    "seqavg",
-]
+MULTI_PARTNER_LEARNING_APPROACHES = {
+    "fedavg": multi_partner_learning.FederatedAverageLearning,
+    "seq-pure": multi_partner_learning.SequentialLearning,
+    "seq-with-final-agg": multi_partner_learning.SequentialWithFinalAggLearning,
+    "seqavg": multi_partner_learning.SequentialAverageLearning
```

Comment:
We should stick with snake case here I think.

Reply:
For the keys or the values ? The values are classes, so I used CamelCase. And for the keys I copied the current parameters

Reply:
- For class names PEP8 recommends CapWords, so that is good.
- For the names of the multi-partner learning approaches, I think @RomainGoussault you mean use underscores instead of dashes, right? I don't know. I liked the idea of marking the differences between variables and functions names (snake_case), and "actual names".

Reply:
I'd rather go for snake case, but keeping method names like this is ok for me